sys/net/gnrc/netif/Kconfig

```diff
@@ -22,12 +22,12 @@ config GNRC_NETIF_MSG_QUEUE_SIZE_EXP
 
 config GNRC_NETIF_IPV6_ADDRS_NUMOF
     int "Maximum number of unicast and anycast addresses per interface"
-    default 2
+    default 3
     help
         If you change this, please make sure that
         @ref CONFIG_GNRC_NETIF_IPV6_GROUPS_NUMOF is also large enough to fit the
         addresses' solicited nodes multicast addresses.
-        Default: 2 (1 link-local + 1 global address).
+        Default: 3 (1 link-local + 2 global addresses).
```

Comment:
I guess these changes should be replaced with a the possibility to configure `GNRC_NETIF_IPV6_DHCPV6_IA_NA_ADDR`?

Reply:
If you use `CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX` to define the maximum number of addresses instead I think you can use that to define a conditional default.